keras/models.py

```diff
@@ -41,8 +41,19 @@ def slice_X(X, start=None, stop=None):
         else:
             return X[start:stop]
 
-def calculate_class_weights(Y, class_weight):
-    if isinstance(class_weight, dict):
+def standardize_instance_weights(instance_weight):
+    if isinstance(instance_weight, list):
+        w = np.array(instance_weight)
+    elif isinstance(instance_weight, np.ndarray):
+        w = instance_weight
+    else:
+        w = None
```

Comment:
Is `None` viable? If not, shouldn't we throw an exception instead of returning `None`?